web/react/components/user_settings/custom_theme_chooser.jsx

```diff
@@ -19,6 +19,15 @@ export default class CustomThemeChooser extends React.Component {
         });
         $('.color-picker').on('changeColor', this.onPickerChange);
     }
+    componentDidUpdate() {
```

Comment:
So that anytime the theme changes, no matter how, the colorpickers will match for sure. If it was just done on the paste event, then if the input box for a color was changed another way then the colorpicker wouldn't update properly.

This is also more React-like, as before the colorpickers were not tied to the theme prop (they have their own internal state). This way their internal state is always updated on render to match whatever the theme prop has.